src/EditorFeatures/CSharpTest/EditAndContinue/ActiveStatementTests.cs

```diff
@@ -622,6 +622,28 @@ static void Foo(int a)
             edits.VerifyRudeDiagnostics(active);
         }
 
+        [WpfFact]
+        public void Delete_Entire_Namespace()
```

Comment:
Test names are usually in PascalCase, though you will occasionally see underscores before suffixes that distinguish the member of a family of tests (Overriding_Parent vs Overriding_Grandparent).